### PR TITLE
Configure Spring

### PR DESCRIPTION
**Pull request by pavlokhomiak**

### CODE REVIEW

File: src/main/resources/db.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#MySQL properties
+db.driver=com.mysql.cj.jdbc.Driver
+db.url=jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC
+db.username=root
+db.password=fragolino21023
+
+#Hibernate properties
+hibernate.show_sql=true
+hibernate.hbm2ddl.auto=create-drop
```
Comment (ilya449) [inline src/main/resources/db.properties]: empty line

**Review by ilya449 — APPROVED** [review]

File: src/main/java/com/dev/cinema/config/AppConfig.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.dao.impl",
+        "com.dev.cinema.service.impl",
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/config/AppConfig.java]:
```suggestion
        "com.dev.cinema.dao",
        "com.dev.cinema.service",
```

File: src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java [hunk 3]
```diff
@@ -2,25 +2,31 @@
...
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
+import org.springframework.stereotype.Repository;
 
-@Dao
+@Repository
 public class CinemaHallDaoImpl implements CinemaHallDao {
     private static final Logger logger = Logger.getLogger(CinemaHall.class);
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java]:
```suggestion
    private static final Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
```

File: src/main/java/com/dev/cinema/service/impl/CinemaHallServiceImpl.java [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 import com.dev.cinema.dao.CinemaHallDao;
-import com.dev.cinema.lib.Inject;
-import com.dev.cinema.lib.Service;
 import com.dev.cinema.model.CinemaHall;
 import com.dev.cinema.service.CinemaHallService;
 import java.util.List;
+import org.springframework.stereotype.Service;
 
 @Service
 public class CinemaHallServiceImpl implements CinemaHallService {
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/service/impl/CinemaHallServiceImpl.java]: we can remove empty line

File: src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java [hunk 5]
```diff
@@ -2,25 +2,27 @@
...
 import java.util.Optional;
 import java.util.logging.Logger;
+import org.springframework.stereotype.Service;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
     private static final Logger logger = Logger.getLogger(Main.class.getName());
 
-    @Inject
     private UserService userService;
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java]: let's remove empty line

File: src/main/java/com/dev/cinema/service/impl/ShoppingCartServiceImpl.java [hunk 6]
```diff
@@ -2,23 +2,24 @@
...
 import com.dev.cinema.model.Ticket;
 import com.dev.cinema.model.User;
 import com.dev.cinema.service.ShoppingCartService;
+import org.springframework.stereotype.Service;
 
 @Service
 public class ShoppingCartServiceImpl implements ShoppingCartService {
-
-    @Inject
     private TicketDao ticketDao;
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/service/impl/ShoppingCartServiceImpl.java]: )

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**